packages/components/src/components/Integration/index.tsx

```diff
@@ -21,15 +20,9 @@ const Header = styled(Stack)(
 
 const Content = styled(Element)(
   css({
-    display: 'grid',
-    gridTemplateColumns: '1fr 50px',
-    gridGap: 2,
-    alignItems: 'center',
-    paddingX: 2,
```

Comment:
Note for reviewer: Had to remove these because the layout changes based on 1) if you're signed in or not 2) list items are 100% width